src/core/models.py

```diff
@@ -270,3 +267,18 @@ def kafka_ready(self) -> bool:
             return False
 
         return True
+
+    def brokers_active(self) -> bool:
+        """Check that Kafka is active."""
+        # FIXME SSL connection info through models?
```

Comment:
**question** Can we elaborate more on this? can we maybe have a Jira ticket for this?